docs/ref/models/querysets.txt

```diff
@@ -3903,7 +3903,9 @@ by the aggregate.
 
     * Default alias: ``<field>__avg``
     * Return type: ``float`` if input is ``int``, otherwise same as input
-      field, or ``output_field`` if supplied
+      field, or ``output_field`` if supplied. 
+    * Returns ``None`` for empty querysets or groups.
+
```

Comment:
This isn't quite right - it doesn't return `None`, it returns the value provided in `default`. Also, the "return type" isn't quite correct as it potentially could be `None`.

Perhaps something like the following would be better?

```suggestion
      field, or ``output_field`` if supplied. If the queryset or grouping is
      empty, ``default`` is returned, which is ``None`` by default
```